app/controllers/chargeback_controller.rb

```diff
@@ -305,11 +305,16 @@ def cb_rates_delete
         end
       end
       process_cb_rates(rates, "destroy")  unless rates.empty?
-      add_flash(_("The selected %{records} were deleted") %
-        {:records => ui_lookup(:models => "ChargebackRate")}, :info, true) unless flash_errors?
-      cb_rates_list
-      @right_cell_text = _("%{typ} %{model}") % {:typ => x_node.split('-').last, :model => ui_lookup(:models => "ChargebackRate")}
-      replace_right_cell([:cb_rates])
+      if flash_errors? && @flash_array.count == 1
```

Comment:
Why the check for `@flash_array.count == 1`?  Also note that this implies that the `else` will be called if there are no flash errors **or** `@flash_array.count != 1` (i.e. 0, 2, 3, ...).  Is this the intended behavior?